hw3/01-characters.js

```diff
@@ -1,4 +1,50 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const list = document.getElementById('list');
 
-fetch(url);
+const createCharacterCard = (character) => {
+    const card = document.createElement('div');
+    card.classList.add('card', 'p-1', 'text-center');
+
+    const image = document.createElement('img');
+    image.src = character.imageUrl;
+    image.alt = `${character.image}`;
```

Comment:
This should be a desciption of the image.   Getting warnings for alt text error when Waving the page

Reply:
Updated.

Reply:
I could not add description of the image. So just added fullname for alt tag.

Reply:
For alt text, you can do something like alt=`Picture of ${character.fullName}` . This should get rid of the wave error message